src/System.Linq.Parallel/tests/Combinatorial/CancellationParallelQueryCombinationTests.cs

```diff
@@ -278,6 +278,27 @@ public static void LastOrDefault_OperationCanceledException_PreCanceled(LabeledO
         }
 
         [Theory]
+        [MemberData(nameof(UnaryCancelingOperators))]
+        [MemberData(nameof(BinaryCancelingOperators))]
+        public static void Max_OperationCanceledException(Labeled<Func<CancellationToken, Operation>> source, Labeled<Func<Action, Operation>> operation)
+        {
+            Functions.AssertEventuallyCanceled((token, canceler) => Cancel(token, canceler, source, operation).Max());
```

Comment:
I think I was abusing my knowledge of the internals too much (since it just prepends a `Select` for the conversion).  It being missing does look a little strange.